### PR TITLE
Update sepolia batcher hash

### PR DESCRIPTION
**Pull request by cody-wang-cb**
Updating batcher hash for Sepolia. Similar to the [script](https://github.com/base-org/contract-deployments/pull/207) conducted for sepolia-alpha except in multisig.

Changelog: https://docs.google.com/document/d/1VTGKFdAq3BtoDPQMuH4E47g0t0u9H_fi1eo1RJMFOn8/edit#heading=h.iyrs6mrn1443
Runbook: https://docs.google.com/document/d/1pP2apZYK7kyuKy-e_T9Y-guu_css-JL48xQIe-USkqQ/edit#heading=h.dg6rey1ocmm6

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: sepolia/2024-08-26-update-batcher-hash/.env [hunk 1]
```diff
@@ -0,0 +1,8 @@
+OP_COMMIT=9047beb54c66a5c572784efec8984f259302ec92
+BASE_CONTRACTS_COMMIT=5d98dab6a4f3ba60713a17417a2df7a17d77c52f
+
+# Batcher
+L1_SYSTEM_CONFIG_ADDRESS=0xf272670eb55e895584501d564AfEB048bEd26194
+SYSTEM_CONFIG_OWNER=0x0fe884546476dDd290eC46318785046ef68a0BA9
```
Comment (0x00101010) [inline sepolia/2024-08-26-update-batcher-hash/.env:6]: https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/sepolia/base.toml#L39C24-L39C66

File: sepolia/2024-08-26-update-batcher-hash/.env [hunk 2]
```diff
@@ -0,0 +1,8 @@
+OP_COMMIT=9047beb54c66a5c572784efec8984f259302ec92
+BASE_CONTRACTS_COMMIT=5d98dab6a4f3ba60713a17417a2df7a17d77c52f
+
+# Batcher
+L1_SYSTEM_CONFIG_ADDRESS=0xf272670eb55e895584501d564AfEB048bEd26194
```
Comment (0x00101010) [inline sepolia/2024-08-26-update-batcher-hash/.env:5]: https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/sepolia/base.toml#L52C24-L52C66

File: sepolia/2024-08-26-update-batcher-hash/.env [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+SYSTEM_CONFIG_OWNER=0x0fe884546476dDd290eC46318785046ef68a0BA9
+NEW_BATCH_SENDER=0xfc56e7272eebbba5bc6c544e159483c4a38f8ba3
+ROLLBACK_BATCH_SENDER=0x6cdebe940bc0f26850285caca097c11c33103e47
```
Comment (0x00101010) [inline sepolia/2024-08-26-update-batcher-hash/.env:8]: https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/sepolia/base.toml#L45

File: sepolia/2024-08-26-update-batcher-hash/.env [hunk 4]
```diff
@@ -0,0 +1,8 @@
+OP_COMMIT=9047beb54c66a5c572784efec8984f259302ec92
+BASE_CONTRACTS_COMMIT=5d98dab6a4f3ba60713a17417a2df7a17d77c52f
+
+# Batcher
+L1_SYSTEM_CONFIG_ADDRESS=0xf272670eb55e895584501d564AfEB048bEd26194
+SYSTEM_CONFIG_OWNER=0x0fe884546476dDd290eC46318785046ef68a0BA9
+NEW_BATCH_SENDER=0xfc56e7272eebbba5bc6c544e159483c4a38f8ba3
```
Comment (0x00101010) [inline sepolia/2024-08-26-update-batcher-hash/.env:7]: @cody-wang-cb not sure if this is planned, but we'll need to update `superchain-registry` later for batcher / proposer address

https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/sepolia/base.toml#L43-L45

File: sepolia/2024-08-26-update-batcher-hash/.env (hunk 2 above)
Comment (BrianBland) [reply sepolia/2024-08-26-update-batcher-hash/.env:5]: Just to add to this, let's use a permalink with the latest commit sha, like https://github.com/ethereum-optimism/superchain-registry/blob/c48846ada62602384ac3e9da3a77ae95a59f475f/superchain/configs/sepolia/base.toml#L52C24-L52C66

File: sepolia/2024-08-26-update-batcher-hash/README.md [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+#### 3.2. Validate correctness of the state diff.
+
+Now click on the "State" tab. Verify that:
+
+1. Verify that the nonce is incremented for the Incident Multisig under the "GnosisSafeProxy" at address `0x14536667Cd30e52C0b458BaACcB9faDA7046E056`:
+
+```
+Key: 0x0000000000000000000000000000000000000000000000000000000000000005
+Before: 0x0000000000000000000000000000000000000000000000000000000000000005
+After: 0x00000000000000000000000000000000000000000000000000000000000000006
```
Comment (BrianBland) [inline sepolia/2024-08-26-update-batcher-hash/README.md:200]: We should expect the nonce to increase from 6->7 for the rollback transaction

File: sepolia/2024-08-26-update-batcher-hash/script/RollbackBatcherHash.s.sol [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        return calls;
+    }
+
+    function _ownerSafe() internal view override returns (address) {
+        return SYSTEM_CONFIG_OWNER;
+    }
+
+    function _addOverrides(address _safe) internal view override returns (SimulationStateOverride memory) {
+        IGnosisSafe safe = IGnosisSafe(payable(_safe));
+        uint256 _nonce = _getNonce(safe);
```
Comment (BrianBland) [inline sepolia/2024-08-26-update-batcher-hash/script/RollbackBatcherHash.s.sol:53]: We need to ensure that this nonce is created with the appropriate offset if we're pre-signing this prior to executing the forward update

File: sepolia/2024-08-26-update-batcher-hash/.env (hunk 4 above)
Comment (cody-wang-cb) [reply sepolia/2024-08-26-update-batcher-hash/.env:7]: Yup good call! Created this PR now https://github.com/ethereum-optimism/superchain-registry/pull/526/files

**Review by BrianBland — APPROVED** [review]

**Pull request merged**